build-settings/rollup.config.mjs

```diff
@@ -27,16 +27,17 @@ const createConfig = (pkgName) => {
             },
         ],
         input: `packages/${pkgName}/src/index.ts`,
-        // We're using `builtIns: "usage"` with @babel/preset-env.
-        // This results in individual modules being imported from
-        // core-js directly.  `autoExternal` doesn't know how to
-        // deal with this so we manually externalize these imports.
-        external: [/core-js/],
         plugins: [
             swc({
                 swc: {
                     swcrc: true,
                     minify: true,
+                    // We do _not_ specify "env" here (a la @babel/preset-env)
+                    // because our TypeScrip compiler "target" is set to ES2021
```

Comment:
super-nit(typo):
```suggestion
                    // because our TypeScript compiler "target" is set to ES2021
```

Thanks for adding this comment! :)